src/parser/priest/holy/modules/features/HolyPriestHealingEfficiencyTracker.js

```diff
@@ -68,6 +77,25 @@ class HolyPriestHealingEfficiencyTracker extends HealingEfficiencyTracker {
 
     return spellInfo;
   }
+
+  addEcho(spellInfo, healingSpellIds) {
+    if (this.echoOfLight.masteryHealingBySpell[spellInfo.spell.id]) {
+      spellInfo.healingDone += this.echoOfLight.masteryHealingBySpell[spellInfo.spell.id].effectiveHealing;
+      spellInfo.overhealingDone += this.echoOfLight.masteryHealingBySpell[spellInfo.spell.id].overHealing;
+    }
+
+    if (healingSpellIds) {
+      for (const healingSpellId in healingSpellIds) {
```

Comment:
https://github.com/WoWAnalyzer/WoWAnalyzer/wiki/Meta:-Style-guide
`Please don't use for() if it can be done with array.forEach or array.map.`